openid-sharedsignals-framework-1_0.md

```diff
@@ -2155,9 +2143,9 @@ The purpose is defense in depth against confusion with other JWTs, as described
 in Sections 4.5 and 4.6 of {{RFC8417}}.
 
 ### The "aud" Claim {#aud-claim}
-The "aud" claim can be a single value or an array. Each value SHOULD be the
-OAuth 2.0 client ID. Other values that uniquely identifies the Receiver to the
-Transmitter MAY be used, if the two parties have agreement on the format.
+The "aud" claim can be a single value (string) or an array of strings. Values that 
```

Comment:
"...can be a string or an array of strings."

I don't think "single value" is needed. Alternative would be "a single string or an array of strings"